### ISSUE TITLE
SyncPlay broken

### ISSUE THREAD

**Issue by CaveMedia** [jellyfin/jellyfin#6291]
**Describe the bug**
Everytime someone wants to play something inside a SyncPlay Group, nothing happens. Without being in the SyncPlay Group the content starts instantly.

**System (please complete the following information):**
 - OS: Debian 10
 - Virtualization: Dedicated
 - Clients: Browser
 - Browser:  Firefox and Chrome (latest)
 - Jellyfin Version: 10.7.6
 - Playback: --
 - Installed Plugins: SkinManager, TheTVDB
 - Reverse Proxy: nginx behind Cloudflare (but tested it without already too)
 - Base URL: /
 - Networking: [e.g. Host, Bridge/NAT]
 - Storage: Cloud

**To Reproduce**

1. Open a SyncPlay Group
2. Join the Group from another location
3. Try to click on play

**Expected behavior**
The Content should start normally as it does without being in the group.

**Server Logs**
There are no related logs besides of the join messages

**FFmpeg Logs**
--

**Browser Console Logs**
**GET** https://watch.URL/Items/3bbcc286b699a2191a52e3c2cccca6ec
_[HTTP/2 405 Method Not Allowed 21ms]_

Uncaught (in promise) 
Response { type: "basic", url: "https://watch.URL/Items/3bbcc286b699a2191a52e3c2cccca6ec", redirected: false, status: 405, ok: false, statusText: "Method Not Allowed", headers: Headers, body: ReadableStream, bodyUsed: false }

**Screenshots**
https://i.imgur.com/COP240S.png

**Additional context**
--

**Comment by ikomhoog** [comment]
Playing the video first and then creating the SyncPlay group is a workaround.

**Comment by CaveMedia** [comment]
Alright that actually works, but what's wrong with the syncplay? I mean back then it worked perfectly. Is that a known issue?

**Comment by ikomhoog** [comment]
I did some more searching, First of all, it's not fixed in the Unstable branch
Which most likely means that it's not a known issue.
I downgraded until it worked again, version 10.7.2 works while 10.7.3 doesn't.

I also notice that this is indeed the same log that you have mentioned:
```
main.ec4eecc6298d89e83759.bundle.js:2 Requesting url without automatic networking: https://jellyfin.mydomain.com/Items/dc82eb7a77ff587fe3c189afcbbda9bf
main.ec4eecc6298d89e83759.bundle.js:2 GET https://jellyfin.mydomain.com/Items/dc82eb7a77ff587fe3c189afcbbda9bf net::ERR_ABORTED 405
b @ main.ec4eecc6298d89e83759.bundle.js:2
value @ main.ec4eecc6298d89e83759.bundle.js:2
value @ main.ec4eecc6298d89e83759.bundle.js:2
value @ main.ec4eecc6298d89e83759.bundle.js:2
d @ main.ec4eecc6298d89e83759.bundle.js:2
value @ main.ec4eecc6298d89e83759.bundle.js:2
value @ main.ec4eecc6298d89e83759.bundle.js:2
p @ main.ec4eecc6298d89e83759.bundle.js:2
v @ main.ec4eecc6298d89e83759.bundle.js:2
Ct @ main.ec4eecc6298d89e83759.bundle.js:2
/web/#!/home.html:1 Uncaught (in promise) Response {type: "basic", url: "https://jellyfin.mydomain.com/Items/dc82eb7a77ff587fe3c189afcbbda9bf", redirected: false, status: 405, ok: false, …}
Promise.then (async)
value @ main.ec4eecc6298d89e83759.bundle.js:2
value @ main.ec4eecc6298d89e83759.bundle.js:2
p @ main.ec4eecc6298d89e83759.bundle.js:2
v @ main.ec4eecc6298d89e83759.bundle.js:2
Ct @ main.ec4eecc6298d89e83759.bundle.js:2
```


While diving deeper into the logs I noticed that this is the URL that gets used when playing a video on 10.7.2:
```
https://jellyfin.mydomain.com/Users/a0ecedc672bc44e2b3e0ba9aa00c6246/Items/dc82eb7a77ff587fe3c189afcbbda9bf
```

While this is the one used in the current version:
```
https://jellyfin.mydomain.com/Items/dc82eb7a77ff587fe3c189afcbbda9bf
```

I don't know why this is changed(or where for that matter), but I will look into it.

In the meantime, if someone with more experience sees this, feel free to pick this up and don't wait on me.

**Comment by Break-Off** [comment]
Same issue here. Syncplay works correctly on LAN, but when accesing from an external network, it keeps buffering without playing. Workaround described above doesn't work. Here are my specs:

**System (please complete the following information):**

    * OS: Arch Linux

    * Virtualization: Official Docker Image

    * Clients: Browser

    * Browser:  Firefox and Chrome (latest)

    * Jellyfin Version: 10.7.6

    * Playback: --

    * Installed Plugins: --

    * Reverse Proxy: nginx behind Cloudflare

    * Base URL: /

    * Networking: [e.g. Host, Bridge/NAT]

    * Storage: Local

**Comment by ikomhoog** [comment]
@Break-Off that is an entirely different issue, we are talking about syncplay not playing a video while in a group. not about it being available externally.

I do however know what is probably causing it, when syncplay first arrived I had to do something in nginx to make it work.
The loading bars in my dashboard also didn't update automatically:
![image](https://user-images.githubusercontent.com/9083045/125939088-bebe7972-8964-441c-88a3-52b577485541.png)

I don't remember my exact difference but my Nginx file was different from what the documentation mentioned and it was something with the sockets.
Please check if your nginx config differs from the documentation.
Here is the documentation: https://jellyfin.org/docs/general/networking/nginx.html

### If it doesn't work, please make a separate issue so we don't misuse this one.

**Comment by Break-Off** [comment]
@ikomhoog I would say it's the same issue since using Jellyfin locally means bypassing the nginx proxy, thus adding weight to the proxy being the problem. OP mentions their storage as being cloud-based, possibly meaning they don't have physical access to the server, while I do (my LAN).

Back to the issue, I'm using [SWAG](https://github.com/linuxserver/docker-swag) as my nginx-based reverse-proxy solution, and their [Jellyfin example configuration](https://github.com/linuxserver/reverse-proxy-confs/blob/master/jellyfin.subdomain.conf.sample) is mostly identical to the one you linked except for the /web block (which says it's for aesthetics) and that all security headers and certificates are externalized in two files ([proxy.conf](https://github.com/linuxserver/docker-swag/blob/master/root/defaults/proxy.conf) and [ssl.conf](https://github.com/linuxserver/docker-swag/blob/master/root/defaults/ssl.conf)).

**Comment by sigaloid** [comment]
Having this issue on **10.7.5** (old version) on Docker, via apache2 (default cfg). Both users externally accessing it through the proxy. The workaround didn't work.

**Comment by ikomhoog** [comment]
@Break-Off and @sigaloid If the workaround doesn't work its a different issue.
My server is also on LAN and i have this with and without proxy.

Yes these issues might be related, but they are different.
For you it works on LAN but keeps buffering when accessing externally.
We can not start anything once we are already in a group, local or through a proxy, it does not even start buffering like it does for you, there is just nothing at all.



Please make a separate issue so you don't clutter this one.

**Comment by sigaloid** [comment]
No, my issue is exactly as you described. I cannot start any syncplay media even if I VPN into the server so i am local to it. Apologies for the confusion

**Comment by matthew-jones-uk** [comment]
Same problem, cannot play media with syncplay group.

**Comment by OancaAndrei** [comment]
I can reproduce this as well, with client and server both on the same machine. I'm currently investigating the cause.

Seems related to how the `ApiClient` gets reused: `GET http://localhost:8096/jellyfin/Items/[ID]` should instead be `GET http://localhost:8096/Users/[ID]/Items/[ID]`.
